google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddSmartCampaign.java

```diff
@@ -600,14 +601,20 @@ private MutateOperation createAdGroupAdOperation(
     // Adds additional headlines + descriptions if we didn't get enough back from the suggestion
     // service.
     if (adSuggestions.getHeadlinesCount() < NUM_REQUIRED_HEADLINES) {
-      for (int i = 0; i < NUM_REQUIRED_HEADLINES - adSuggestions.getHeadlinesCount(); ++i) {
+      for (int i = 0;
+          i < NUM_REQUIRED_HEADLINES - adBuilder.getSmartCampaignAdBuilder().getHeadlinesCount();
+          ++i) {
         adBuilder
             .getSmartCampaignAdBuilder()
             .addHeadlines(AdTextAsset.newBuilder().setText("Placeholder headline " + i).build());
       }
     }
     if (adSuggestions.getDescriptionsCount() < NUM_REQUIRED_DESCRIPTIONS) {
-      for (int i = 0; i < NUM_REQUIRED_DESCRIPTIONS - adSuggestions.getDescriptionsCount(); ++i) {
+      for (int i = 0;
+          i
```

Comment:
The format is a bit strange?

Reply:
I know - I checked before and rechecked now, this is google-java-format!